index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <header class="heading">Hello microverse</header>
+</body>
```

Comment:
As part of the [Project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) the text "Hello Microverse!". should be an `h1` tag. e.g 👇 

`<h1 class="heading">Hello microverse</h1>`

please kindly fix that.

H1 tags are an important part of SEO. All of the important pages on your site should have H1 tags to draw in the reader and give a clear indication of the content on the page.